src/20_Components/amp-bind.html

```diff
@@ -79,6 +88,25 @@
 
   `[property]="expression"`
 
+-->
+
+<!-- #### Text for textContent -->
+<!--
+The text of this &lt;p&gt; will change when the `currentAnimal` variable
```

Comment:
It doesn't make sense to add 'either the `allAnimals` or '. because the expression is only using currentAnimal variable.